config_test.go

```diff
@@ -48,16 +48,16 @@ repos:
 	want := &Config{
 		CacheDir: `/tmp`,
 		Port:     9139,
-		Repos: map[string]Repo{
-			"archlinux": Repo{
+		Repos: map[string]*Repo{
+			"archlinux": &Repo{
 				URL: "http://mirrors.kernel.org/archlinux",
 			},
 		},
 		PurgeFilesAfter: 2592000,
 		DownloadTimeout: 200,
 		Prefetch:        &RefreshPeriod{Cron: "0 0 3 * * * *", TTLUnaccessed: 5, TTLUnupdated: 200},
 	}
-	if !cmp.Equal(*got, *want, cmpopts.IgnoreFields(Config{}, "Prefetch")) {
+	if !cmp.Equal(*got, *want, cmpopts.IgnoreFields(Config{}, "Prefetch"), cmpopts.IgnoreUnexported(Repo{})) {
```

Comment:
Well, this will ignore the repo field, right? It is not what I would expect from a test that has to check if configs are matching. I am a bit perplexed from this edit

Reply:
I believe it only ignores the new unexported fields on the `Repo` struct. Apologies if i'm mistaken, i haven't used `cmpopts` before

Reply:
AFAIK it would not check that the Repo pointers addresses match, which is fine. However, the content pointed from the Repo pointers should be checked if they are not both nil, right?